alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -123,11 +123,32 @@ The mitigation depends on which resources are being rejected and why.
     secret key reference in the `ServiceMonitor` or `PodMonitor`
     configuration.
 
+When the alert is triggered by an object managed by a 3rd-party operator, it
```

Comment:
nit: Let’s choose between `object` and `resource` and stick to one term for consistency?